### ISSUE TITLE
unknown items in the elepower tome

### ISSUE THREAD

**Issue by zhaolinlau** [sirrobzeroone/elepower#2]
some machines showing unknown item, it's bug i think? detail please refer to the link below(screenshot)
https://ibb.co/27VbswR

**Comment by sirrobzeroone** [comment]
Yep they do :) - its on my too do list most need icons selected and tool tips.
